files/start-strongswan.sh

```diff
@@ -49,25 +88,37 @@ _term() {
 }
 
 _set_default_variables() {
-    export IPSEC_REMOTEIP=${IPSEC_REMOTEIP:-%any}
-    export IPSEC_LOCALIP=${IPSEC_LOCALIP:-%any}
+    # local and remote IP can not be "%any" if VTI needs to be created
+    if [ -z $IPSEC_VTI_KEY ]
```

Comment:
please add quotes. (it is legal when the var is empty, yes - but it becomes problematic with invalid values)

Reply:
done